pyglmnet/utils.py

```diff
@@ -72,7 +73,20 @@ def log_likelihood(y, yhat, distr):
 
 def tikhonov_from_prior(PriorCov, n_samples):
     """Given a prior covariance matrix, returns a Tikhonov matrix"""
+    """Given a prior covariance matrix, returns a Tikhonov matrix"""
     [U, S, V] = np.linalg.svd(PriorCov, full_matrices=False)
-    Tau = np.dot(np.diag(1. / np.sqrt(S)), U)
-    Tau = 1. / np.sqrt(np.float(n_samples)) * Tau / Tau.max()
+
+    S_ratio = S / S.max()
+    threshold = 0.0001
```

Comment:
thanks! good eye! have made it a parameter now and defined it within the function. it specifies a cutoff of where to truncate singular values based on the ratio of largest to smallest singular value. 

same thing is done in MNE :)